apps/framework-docs-v2/FLAGS_README.md

```
@@ -40,3 +65,6 @@
 - `src/flags.ts` - Flag definitions
 - `src/app/.well-known/vercel/flags/route.ts` - Discovery endpoint
 
+## Related Documentation
+
+- `GUIDES_README.md` - How to write and promote guides
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)